<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>barWindowClass</class>
 <widget class="QMainWindow" name="barWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>色标编辑窗</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.ico</normaloff>icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>999999</width>
     <height>999999</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="nowLabel">
        <property name="text">
         <string>样式名称：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nowLine">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="BarAdvance" name="bar" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>75</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item row="3" column="2">
       <widget class="QLineEdit" name="SLine">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="IndexLabel">
        <property name="text">
         <string>当前关键点序号:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="IndexNumLabel">
        <property name="text">
         <string>关键点总数:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="GLabel">
        <property name="text">
         <string>G:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="BLabel">
        <property name="text">
         <string>B:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="RLabel">
        <property name="text">
         <string>R:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="HLabel">
        <property name="text">
         <string>H:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="SLabel">
        <property name="text">
         <string>S:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="VLabel">
        <property name="text">
         <string>V:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="GLine">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="RLine">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="IndexLine">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="HLine">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="BLine">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="VLine">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="alphaLabel">
        <property name="text">
         <string>透明度：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="alphaLine">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="IndexNumLine">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>差值空间：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>RGB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HSV</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="reversalButton">
        <property name="text">
         <string>色标反转</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>224</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="warning">
        <property name="text">
         <string>注意：预置色标禁止修改！</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="OKButton">
        <property name="text">
         <string>修改</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="AddButton">
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CancelButton">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>709</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BarAdvance</class>
   <extends>QWidget</extends>
   <header>BarAdvance.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="barwindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>OKButton</sender>
   <signal>clicked()</signal>
   <receiver>barWindowClass</receiver>
   <slot>OK()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>552</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CancelButton</sender>
   <signal>clicked()</signal>
   <receiver>barWindowClass</receiver>
   <slot>Cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>625</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>636</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AddButton</sender>
   <signal>clicked()</signal>
   <receiver>barWindowClass</receiver>
   <slot>Add()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>572</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked()</signal>
   <receiver>barWindowClass</receiver>
   <slot>Delete()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>523</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>534</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>barWindowClass</receiver>
   <slot>InterpolationChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reversalButton</sender>
   <signal>clicked()</signal>
   <receiver>barWindowClass</receiver>
   <slot>Reversal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>597</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>562</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OK()</slot>
  <slot>Cancel()</slot>
  <slot>Add()</slot>
  <slot>Delete()</slot>
  <slot>InterpolationChange()</slot>
  <slot>Reversal()</slot>
 </slots>
</ui>
